### ISSUE TITLE
Help /w Limit User stream count, kill last stream

### ISSUE THREAD

**Issue by TsinJu** [blacktwin/JBOPS#429]
Sry but did not find any help to my Question

In the readme, it's explained to use  "Triggers: Playback Start" for killing more streams than e.g. 3

Wouldn't "User Concurrent Streams" do the same?

What I want to do is to Limit a User to 1 Stream

**Comment by blacktwin** [comment]
Probably.

**Comment by TsinJu** [comment]
Yeah whatever, working on my own limiter now since the killstream isn't working correctly as it also recon fast forward or rewind or quality changes sometimes as another stream.... Anyway

**Comment by blacktwin** [comment]
> killstream isn't working correctly as it also recon fast forward or rewind or quality changes sometimes as another stream

I'd be interested in knowing more about this and know what changes you've made or your approach in your limiter.

**Comment by TsinJu** [comment]
Will send it, after I'm done. It also has integrated Discord Message that's been sent via Webhook.
Limits will be set into an ini. Makes it easier to handle Users limits.

Nothing to complicated.... But I'm not done yet. Let you know asap
